images/linux/scripts/installers/edge.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+################################################################################
+##  File:  edge.sh
+##  Desc:  Installs Edge
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/install.sh
+
+# Import Microsoft Edge GPG Key and Repository
+wget -O- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor | tee /usr/share/keyrings/microsoft-edge.gpg
```

Comment:
nit: we tend to use redirection instead of tee
```suggestion
wget -O- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /usr/share/keyrings/microsoft-edge.gpg
```